xsql/parser_test.go

```diff
@@ -1205,21 +1205,9 @@ func TestParser_ParseStatement(t *testing.T) {
 		},
 
 		{
-			s: `SELECT sample(-.3,) FROM tbl`,
-			stmt: &SelectStatement{
-				Fields: []Field{
-					{
-						Expr: &Call{
-							Name: "sample",
-							Args: []Expr{
-								&NumberLiteral{Val: -0.3},
-							},
-						},
-						Name:  "sample",
-						AName: ""},
-				},
-				Sources: []Source{&Table{Name: "tbl"}},
-			},
+			s:    `SELECT sample(-.3,) FROM tbl`,
```

Comment:
Why this statement is returned with nil value?

Reply:
"sample" is not a valid function name. There is no function validation previously for functions with no arguments so that it passed the parser. The validation for no argument functions is added in this PR.